### DISCUSSION TITLE
Enhancements process improvements

### DISCUSSION THREAD

**Discussion by savitharaghunathan** [konveyor/enhancements#101] [General]
Hi Folks, this is a place to chat about 
1.  What mandatory fields we would like to see on an enhancement proposal

-  For example, have a test plan, API changes, etc (Note: These are just examples)

2.  How can we improve the enhancement process?

Add in your thoughts.

**Comment by shawn-hurley** [comment]
I believe that enhancement MUST have the following:
1. All user-facing changes from an API perspective 
2. Upgrade and Downgrade strategy

I personally don't care as much at getting the UI changes but maybe that does make sense as well.

**Comment by savitharaghunathan** [comment]
I would like to see a section on UX for user facing changes. It can be just a line or two on the impact that this enhancement would have on the UX.
